viewers/nuxeo-video-viewer.html

```diff
@@ -38,27 +39,26 @@
 
       #video {
         background: var(--primary-background-color);
-        height: 100%;
+        max-height: 100%;
         width: 100%;
         outline: none;
         @apply --layout-flex;
       }
 
       #video.hasStoryboard {
-        height: calc(100% - 85px);
+        max-height: calc(100% - 85px);
       }
 
       #storyboard {
-        display: block;
         background: var(--primary-background-color);
+        display: grid;
         height: 85px;
```

Comment:
Also, move this to the `.thumbnail` class, to prevent it from changing on firefox.